### DISCUSSION TITLE
Show only first 3 letters of a month, how to do it?

### DISCUSSION THREAD

**Discussion by GenatHadri** [gpbl/react-day-picker#1783] [Support] [answered]
Hello everyone,
I am using the react day picker library, but I want somehow to show only first 3 letters of a month, how to do it?

**Comment by gpbl** [comment, marked as answer]
Hi @GenatHadri! Where do you want to show the first 3 letters? If it is in the caption, you can pass a `formatCaption` function to the [formatters prop](https://react-day-picker.js.org/api/types/Formatters):

```tsx
<DayPicker
  formatters={{
    formatCaption: (date) => format(date, "MMM yyyy").toUpperCase()
  }}
/>
```

Example: https://codesandbox.io/s/react-daypicker-forked-djmu70?file=/src/App.tsx
